### PR TITLE
Include .wrangler directory in gitignore template

### PR DESCRIPTION
**Pull request by Cyb3r-Jak3**
What this PR solves / how to test:

Add the `.wrangler` directory to the `gitignore` template.

Author has included the following, where applicable:

- [ ] ~~Tests~~
- [X] Changeset

Reviewer has performed the following, where applicable:

- [ ] Checked for inclusion of relevant tests
- [ ] Checked for inclusion of a relevant changeset
- [ ] Checked for creation of associated docs updates
- [ ] Manually pulled down the changes and spot-tested

### CODE REVIEW

File: .changeset/gold-laws-sleep.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+feature: include .wrangler directory in gitignore template
```
Comment (petebacondarwin) [inline .changeset/gold-laws-sleep.md:5]:
```suggestion
chore: include .wrangler directory in gitignore template
```

I think we can avoid the feature marker here. It is more of an internal tweak.
I might go so far as to suggest it does not need a changeset....

**Review by petebacondarwin — APPROVED** [review]
LGTM - thanks.

File: .changeset/gold-laws-sleep.md (hunk 1 above)
Comment (IgorMinar) [reply .changeset/gold-laws-sleep.md:5]: Isn't this a publicly observable behavior change? If so, why wouldn't we want a changeset?

Comment (petebacondarwin) [reply .changeset/gold-laws-sleep.md:5]: It is only noticeable for people who are contributing to Wrangler. I.e. if they run a template from within the working directory of the workers-sdk repository.

Comment (Cyb3r-Jak3) [reply .changeset/gold-laws-sleep.md:5]: This is a public change. This template is use when you choose git to manage a worker during init. 
https://github.com/cloudflare/workers-sdk/blob/2b641765975d98e6e04342533fa088f51a96acab/packages/wrangler/src/init.ts#L237

Comment (petebacondarwin) [reply .changeset/gold-laws-sleep.md:5]: Oh!! So sorry. I thought this change was in `/templates/.gitignore`, not `/packages/wrangler/templates/.gitignore.`.

Please ignore my comments!

Or actually, can you make it clear in the changeset that this affects the output of generating a new worker using `wrangler init`?

Comment (Cyb3r-Jak3) [reply .changeset/gold-laws-sleep.md]: Done

**Review by JacobMGEvans — APPROVED** [review]

**Pull request merged**